.gitlab/e2e/e2e.yml

```diff
@@ -288,6 +288,8 @@ new-e2e-installer:
     - deploy_deb_testing-i7_amd64
     - deploy_rpm_testing-i7_arm64
     - deploy_rpm_testing-i7_x64
+    - deploy_installer_oci
+    - deploy_agent_oci
```

Comment:
Not used now but will be useful when testing the download of the commit's agent OCI